app.js

```diff
@@ -1,173 +1,102 @@
-  hourlySales: [],
-  totals: 0,
-  grandTotal: 0,
-  customerPerHour: function () {
+var pikePlace = new Store('Pike Place', 17, 88, 5.2);
+var seaTacAirport = new Store('SeaTac Airport', 6, 24, 1.2);
+var Southcenter = new Store('Southcenter', 11, 38, 1.9);
+var bellevueSquare = new Store('Bellevue Square', 20, 48, 3.3);
+var alki = new Store('Alki', 3, 24, 2.6);
+
+function Store(name, min, max, avgSales) {
+  this.name = name;
+  this.min = min;
+  this.max = max;
+  this.avgSales = avgSales;
+  this.hourlySales = [];
+  this.totals = 0;
+}
+
+Store.prototype.customerPerHour = function(min, max) {
   return Math.random() * (this.max - this.min) + this.min;
-},
-  generateHourly: function(arr, totals, avg, rand, hour) {
-    for(var j = 0; j < hour.length; j++) {
-        var ran = Math.floor(this.customerPerHour(this.max, this.min) * avg);
-        arr.push(ran);
-        this.totals += ran;
-        }
-},
-  render: function() {
-    this.generateHourly(this.hourlySales, this.totals, this.avgSales, this.customerPerHour, hours);
-    var ulEl = document.createElement('ul');
-    ulEl.appendChild(document.createTextNode(this.name));
-    var sectionEl = document.getElementById('pikePlace').appendChild(ulEl);
-
-    for(var i = 0; i < hours.length; i++) {
-      var liEl = document.createElement('li');
-      liEl.textContent = hours[i] + ": " + this.hourlySales[i];
-      ulEl.appendChild(liEl);
-    }
-    var liEl_two = document.createElement('li');
-    liEl_two.textContent = 'Total: ' + this.totals;
-    ulEl.appendChild(liEl_two);
+};
+
+Store.prototype.generateHourly = function(arr, totals, avg, ran, hour) {
+  for(var j = 0; j < hours.length; j++) {
+    var ran = Math.floor(this.customerPerHour(this.max, this.min) * this.avgSales);
+    this.hourlySales.push(ran);
+    this.totals += ran;
   }
-},
-{
-  name: 'SeaTac Airport',
-  min: 6,
-  max: 24,
-  avgSales: 1.2,
-  hourlySales: [],
-  totals: 0,
-  customerPerHour: function () {
-    return Math.random() * (this.max - this.min) + this.min;
-  },
-  generateHourly: function(arr, totals, avg, rand, hour) {
-    for(var j = 0; j < hours.length; j++) {
-      var ran = Math.floor(this.customerPerHour(this.max, this.min) * avg);
-      arr.push(ran);
-      this.totals += ran;
-    }
-  },
-  render: function() {
-    this.generateHourly(this.hourlySales, this.totals, this.avgSales, this.customerPerHour, hours);
-    var ulEl = document.createElement('ul');
-    ulEl.appendChild(document.createTextNode(this.name));
-    var sectionEl = document.getElementById('seaTacAirport').appendChild(ulEl);
-
-    for(var i = 0; i < hours.length; i++) {
-      var liEl = document.createElement('li');
-      liEl.textContent = hours[i] + ": " + this.hourlySales[i];
-      ulEl.appendChild(liEl);
-    }
-    var liEl_two = document.createElement('li');
-    liEl_two.textContent = 'Total: ' + this.totals;
-    ulEl.appendChild(liEl_two);
+};
+
+var tableEl = document.getElementById('stores');
```

Comment:
All of the code here should be scoped inside a function, which is then called somewhere. Perhaps put it in an IIFE that runs on page load?